lib/whatsapp_sdk/api/client.rb

```diff
@@ -83,6 +97,7 @@ def faraday(url:, multipart: false)
           client.request(:url_encoded)
           client.adapter(::Faraday.default_adapter)
           client.headers['Authorization'] = "Bearer #{@access_token}" unless @access_token.nil?
+          client.response(:logger, @logger, @logger_options) unless @logger.nil?
```

Comment:
do we need any validation on `logger_options`?

Reply:
I'm fine with the latest update on the README, thanks!

Reply:
As long as it's a `Hash` , we are good to use it here. Its sorbet validation is taking care of it.

Moreover, `logger`'s validation is here because if nil is passed for the logger, then Faraday assigns the Standard Output for the logger.
This check makes sure that only configure the logger if the user is deliberately configuring it. Otherwise, not.

Please let me know if you have any follow-ups. Thanks. :)